README.md

```diff
@@ -132,13 +134,19 @@ An alternative is to use the fixtures:
 configure_beaker(modules: :fixtures)
 ```
 
-This will switch to use [puppet-modulebuilder](https://github.com/puppetlabs/puppet-modulebuilder) on all modules present in `spec/fixtures/modules`. This is faster, but more importantly it also allows using git versions of modules. No dependency resolution is done and it is up to the module developer to ensure it's a correct set. It is also up to the module developer to ensure the fixtures are checked out before beaker runs.
+This will switch to use [puppet-modulebuilder](https://github.com/puppetlabs/puppet-modulebuilder) on all modules present in `spec/fixtures/modules`.
+This is faster, but more importantly it also allows using git versions of modules.
+No dependency resolution is done and it is up to the module developer to ensure it's a correct set.
+It is also up to the module developer to ensure the fixtures are checked out before beaker runs.
 
 ```ruby
 # In Rakefile
-task :beaker => "spec_prep"
+task :beaker => "fixtures:prep"
 ```
 
+The `fixtures:prep` task is provided by the [puppet_fixtures](https://github.com/voxpupuli/puppet_fixtures/blob/master/lib/puppet_fixtures/tasks.rb) gem.
+voxpupuli-acceptance automatically configures it for you.
```

Comment:
Not really because you have the line in `Rakefile` that you need to add. The actual module building is just iterating over what's in the `fixtures/modules` directory.
```suggestion
```